karma.conf.js

```diff
@@ -0,0 +1,69 @@
+// Karma configuration
+// Generated on Sat Apr 01 2017 01:03:22 GMT-0500 (CDT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'src/**/*.spec.js',
```

Comment:
As per the path, I wasn't sure where we wanted tests to live, and I defaulted to separate space, much like the set up in RJL . I'll make this path change and remove phantom.